fcs/fcs.props

```diff
@@ -3,12 +3,5 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
```

Comment:
Why not delete the file since it's empty?

Reply:
done, thanks